docs/pywbemcli/cmdlineinterface.rst

```diff
@@ -383,10 +403,81 @@ shown as the cursor:
     pywbemcli> cl_
                   class
 
+Interactive mode uses a combination of tab completion and auto-suggestion  for
+aut completion which are both always active:
+
+  * ref:`tab-completion` - In this mode, a single <TAB> enables the display of
+    available completion possibilities for command groups, commands, options
+    and selected option values.
+  * ref:`auto-suggestion` - The pywbemcli interactive mode also supports
+    automated parameter suggestions based on the pywbemcli history file which
+    works with the tab-completion described above. The input is compared to
+    the history and when there is another entry starting with the given text,
+    the completion will be shown as gray text behind the current input.
+    Pressing the right arrow → or c-e will insert this suggestion.
```

Comment:
Consistent referencing to special keys, eg `<CTRL>`.

Reply:
DONE